### ISSUE TITLE
HTML Embeds for Dev.to posts

### ISSUE THREAD

**Issue by bnb** [forem/forem#1445]
**Describe the solution you'd like:**
I'd like to see HTML embeds in addition to social sharing, along the lines of how you can embed Tweets or YouTube videos with simple HTML (plus a script or iframe, respectively).

**Describe alternatives you've considered:**
- Adding oembed, Embedly, etc.
  - Isn't viable for every site, especially not ones that I contribute to but don't own.
- Hand coding a pretty embed
  - Not enough of a reason on the user end to do this.

**Additional context:**
Would love to include a pretty dev.to embed in a blog post that is being posted in a couple days but unfortunately it doesn't currently exist.

**Comment by ferdi2005** [comment]
Why don't use this? [Ruby-Oembed](https://github.com/ruby-oembed/ruby-oembed)

**Comment by maestromac** [comment]
not stale

**Comment by jessleenyc** [comment]
Another request here: https://github.com/thepracticaldev/dev.to/issues/3676

**Comment by jitendra1998** [comment]
Hi @jessleenyc and @maestromac, interested in working on this feature.

**Comment by jitendra1998** [comment]
Hi,

Added oembed liquid tag using ruby-oembed gem.
Here is an example of amcharts.com chart in dev.to editor,

![Demo amchart](https://user-images.githubusercontent.com/13114810/79053398-b4c60a80-7c5a-11ea-9519-3d06f621f422.png)

A related issue for the ability to add charts in dev.to [#6362](https://github.com/thepracticaldev/dev.to/issues/6362)

**Comment by cmgorton** [comment]
Hello! Thanks for your contributions here.  As a heads up, I am going to close this issue.  
We have a new process moving forward to address feature requests and especially those that have been in our repo for 1+ years without much traction.  
**TLDR**: we have new internal RFC process for feature requests. Any features that are more substantial and require more thought and detail will need to be discussed in [forem.dev](forem.dev).

You can read more about our Internal RFC Process and forem.dev Discussions in [this post](https://forem.dev/foremteam/internal-rfc-process-and-forem-dev-discussions-3gl4).
